README.md

```diff
@@ -67,6 +67,7 @@ module "example" {
 | client_dns_search_domain | The DNS search domain to be pushed to VPN clients. | `string` | n/a | yes |
 | client_dns_server | The address of the DNS server to be pushed to the VPN clients. | `string` | n/a | yes |
 | client_inactive_timeout | The number of seconds of tolerable user inactivity before a client will be disconnected from the VPN. | `number` | `3600` | no |
+| client_motd_url | A URL to the motd page.  This will be pushed to VPN clients as an environment variable. | `string` | n/a | yes |
```

Comment:
Why is this variable required?  What if someone doesn't want to have a MOTD?  Can we safely default to an empty string or something else?

Reply:
It is required because this is part of a required security control.